src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -2292,6 +2292,10 @@ def readCustomCategoryInfo(self):
             plugin_list.append([name, True])
         if plugin_list:
             self.master_category_dict[CATEGORY_CUSTOM] = plugin_list
+        # Adding plugins classified as structure factor to 'CATEGORY_STRUCTURE' list
+        plugin_structure_list = [[name, True] for name, plug in self.custom_models.items() if plug.is_structure_factor]
+        if plugin_structure_list:
+            self.master_category_dict[CATEGORY_STRUCTURE].extend(plugin_structure_list)
```

Comment:
A potential solution (albeit slower) to resolve the issue noted by @pkienzle (code untested):
```suggestion
 plugin_structure_list = [[name, True] for name, plug in self.custom_models.items() if plug.is_structure_factor]
 for plugin_structure in plugin_structure_list:
    if plugin_structure not in self.master_category_dict[CATEGORY_STRUCTURE]:
        self.master_category_dict[CATEGORY_STRUCTURE].extend(plugin_structure)
```